tests/e2e/cucumber/features/journeys/kindergarten.oc10.feature

```diff
@@ -5,7 +5,8 @@ Feature: Kindergarten can use web to organize a day
   So that i'm sure all parents are informed and have the latest information in a easy and secure way
 
   Background:
-    Given "Admin" creates following users using API
+    Given "Admin" logs in
```

Comment:
What about getting/storing admin user access_token in the `beforeAll` hook? Since the server instance remains the same for the whole test run, if the `API_TOKEN=true`, we can do token setup in `beforeAll` - login admin, get and store the token, and logout. This way we don't have to add admin login-logout steps in every feature file.

Reply:
Thanks. 
sometimes Alice (or another user) creates space via api, so I'll leave the creating and store token in the Session -> login

for admin: I add login and logout in the `beforeAll` instead of scenarios

Reply:
token has expiration 300 sec so all tests were failed after 5 min.
I put login admin, get and store the token, and logout to `before` action. 
and delete token in the `after` action

Reply:
can we use the token for renewals instead of re-logins?

Reply:
Also, we have this env config in oCIS: `IDP_ACCESS_TOKEN_EXPIRATION: 300`
https://doc.owncloud.com/ocis/next/deployment/services/s-list/idp.html

Reply:
And for oC10, the token expires in 1hr (if I am not misunderstanding)
https://doc.owncloud.com/server/10.12/admin_manual/configuration/server/security/oauth2.html#access-token-response

Reply:
After discussion with @individual-it, we decided to keep the current implementation. move getting a login token to the `Before all` section in another PR

Reply:
https://github.com/owncloud/web/issues/9083